package/src/frontend/VideoPlayer/prebuffering.js

```diff
@@ -58,19 +64,19 @@ export const prebuffering = function(player) {
         };
 
         timeout();
-        player.prebufferDeferred = deferred;
+        player.prebufferPromise = promise;
       }
     }
 
-    return player.prebufferDeferred ? player.prebufferDeferred.promise() : deferred.resolve().promise();
+    return player.prebufferPromise ? player.prebufferPromise : promise;
```

Comment:
Ok, so the spec for the "simple" success case would also have been worth it after all: In manual testing I noticed that autoplay videos sometimes do not start. This is caused by this line: If the video has already buffered enough right from the start, a pending promise instead of a resolved one is returned now.